### ISSUE TITLE
doc-v3: remove dependency on kvoptions

### ISSUE THREAD

**Issue by FrankMittelbach** [FrankMittelbach/fmitex#18]
V3 currently uses kvoptions.sty but it would be better to use L3 keys instead.

Once this is done the added dependecy in latex2e repo should be taken out again (build.lua and support dir)

**Comment by FrankMittelbach** [comment]
no longer relevant
